CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 1.6.0
+* Ruby 2.3 and 2.4 support dropped.
```

Comment:
[nits] can you use the active tense for consistency? e.g. "Drop support..."

Reply:
sure @jfeltesse-mdsol I will update and merge